packages/editor/src/components/common/topbar/newProject/newProjectDialog/index.tsx

```diff
@@ -44,13 +44,13 @@ class NewProjectDialog extends Component<IProps> {
             <div className={classNames([style.menu, 'contextMenu'])}>
                 <ul>
                     <li>
-                        <div onClick={() => createProjectFromTemplate(Templates.templates[3])}>
-                            Create empty project
+                        <div onClick={this.startWithTemplate}>
```

Comment:
I would make sure we sync the terminology. If we call them examples in the ui, then I think would be best to rename them internally as well.